src/main/webapp/mydata_templates/pagination.html

```diff
@@ -7,15 +7,16 @@
     <div class="col-xs-12 col-md-3">
         <br />
         {% if numResults > 1 %}
-            Found: <b>{{ numResults }} results</b>
+            Found: <b>{{ numResultsString }} results</b>
```

Comment:
Should "pagination.html" continue to live in "mydata_templates"? It seems like it's become more generic and used in other places such as listing users.